### DISCUSSION TITLE
https://www.spsanderson.com/steveondata/posts/2024-11-22/

### DISCUSSION THREAD

**Discussion by giscus** [spsanderson/steveondata#18] [Comments]
# https://www.spsanderson.com/steveondata/posts/2024-11-22/

Discover the power of VI and VIM, the essential text editors for Linux beginners. Master modal editing, navigation, and advanced features to boost your productivity on the command line.

https://www.spsanderson.com/steveondata/posts/2024-11-22/

**Comment by thomascwells** [comment]
Thanks for this!

**Comment by spsanderson** [comment]
You’re welcome

Steven P Sanderson II, MPH
Amazon Author Profile
<https://www.amazon.com/stores/Steven-P.-Sanderson/author/B0CZ157DHK?ref_=pe_2466670_811284380&isDramIntegrated=true&shoppingPortalEnabled=true>
Book on Lulu <http://goo.gl/lmrlFI>
Personal Site <http://www.spsanderson.com>
